### DISCUSSION TITLE
Is it possible to add a text to the toolbar from frontmatter?

### DISCUSSION THREAD

**Discussion by pintergreg** [markmap/markmap#287] [Q&A] [answered]
I would like to add an attribution note to the generated mindmap and the toolbar seems to be a perfect place for it.
I'm using markmap-cli for the conversion, that is why I would like a solution that is available from the markdown input.

**Comment by gera2ld** [comment, marked as answer]
No, the toolbar can only be customized through its API.

**Reply by pintergreg** [reply]
Okay, thanks. 

Could it be a feature? I have a use case, where it would be really convenient for me to set it from the front matter, but do you see it as a feature useful for others?

**Reply by gera2ld** [reply]
I think toolbar actions are too complicated to declare from the frontmatter. It is not even a part of the markmap.
You may easily customize it if you create an HTML and add a piece of JavaScript to it.

Example:
https://github.com/markmap/markmap-vscode/blob/99a8651d685fcc2718134da088e3b30482071808/src/app.ts#L110-L130
